src/Forms/MultiSelectField.php

```diff
@@ -127,7 +127,7 @@ public function loadFrom(DataObjectInterface $record)
         // Detect DB relation or field
         if ($relation instanceof Relation) {
             // Load ids from relation
-            $value = array_values($relation->getIDList() ?? []);
+            $value = $relation->sort(null)->column('ID');
```

Comment:
```suggestion
            $value = $relation->column('ID');
```
Similar to https://github.com/silverstripe/silverstripe-linkfield/pull/405/files#r2299622369 - sort order will determine the order that items appear in the field

Reply:
That link doesn't work (they stop working once the comment is "outdated" - you need to link from outside of the files compare section) but I assume you mean https://github.com/silverstripe/silverstripe-linkfield/pull/405#discussion_r2299622369

Reply:
I can't see any difference in the order of values using `SearchableMultiDropdownField` or `CheckboxSetField` both of which are subclasses of `MultiSelectField` - can you please give me steps to reproduce your findings?